www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+
+The Karmashapes project was initiated to solve the data gap by mashing up several new and existing datasets, like the Indian Ministry of Rural Development’s recently released _Habitations_ open data layer. This work was done in partnership with [Justin Meyers](https://www.linkedin.com/in/justin--meyers/) at Revolutionary GIS.
+
+After a year’s worth of development, we’re excited to announce a step change improvement for India locality coverage in Who’s On First – as openly licensed, CC-BY data.
+
+
+# By the Numbers
+
+We recently finished importing Karmashpes into WOF to generally improve coverage and address common user feedback. Doing so allows Karmashapes to also benefit from WOF’s [stable identifiers](https://github.com/whosonfirst/whosonfirst-properties/blob/main/properties/wof/id.json), [Github](https://github.com/whosonfirst-data/whosonfirst-data/) (as well as Github’s [issue tracking](https://github.com/whosonfirst-data/whosonfirst-data/issues) for future improvements), and to supplement with the [existing sources](https://github.com/whosonfirst/whosonfirst-sources/blob/main/sources/README.md) that WOF already maintains to fill in any coverage or localization gaps that the raw Karmashapes dataset might have.
+
+Importing Karmashapes also allows any downstream consumer of WOF, like [Pelias](https://pelias.io/) via [Geocode.Earth](https://geocode.earth/) to benefit from the full scope of data Karmashapes provides. Let’s take a look at the locality and neighbourhood coverage now available in WOF:
+
+![Locality coverage in Kerala](images/kerala.png "Locality coverage in Kerala")
+_(above) Locality (blue) and neighbourhood (orange) coverage along the northern Kerala coast_
+
+![Locality coverage in Dehradun](images/dehradun.png "Locality coverage in Dehradun")
+_(above) Locality (blue) and neighbourhood (orange) coverage northwest of Dehradun_
+
+![Locality coverage in Chandigarh](images/chandigarh.png "Locality coverage in Chandigarh")
+_(above) Locality (blue) and neighbourhood (orange) coverage in and around Chandigarh_
+
+Above, you can see three areas of India - the northern Kerala coast, areas of Dehradun, and the greater Chandigarh area - with orange representing the neighbourhood placetype and blue representing the locality placetype. These example areas illustrate the complexity, density, and coverage that Karmashapes has added to Who’s On First.
+
+Below are two charts - the first is a per-placetype feature count in WOF before and after the Karmashapes import, the second is a more detailed breakdown of neighbourhood and locality placetype counts per region of India.
```

Comment:
Consider reworking this so its a `(below)` and consolidate with the existing (above) just below the table?

Then the existing "above" (below the table) could turn into a `(below)` for the next table?

Reply:
Below the 2nd chart we need to add:

- `NOTE: Numbers may not completely add up due to partial overlaps with disputed territories._